crates/bevy_pbr/src/meshlet/gpu_scene.rs

```diff
@@ -384,10 +379,18 @@ pub fn prepare_meshlet_per_frame_resources(
                     base_mip_level: i,
                     mip_level_count: Some(1),
                     base_array_layer: 0,
-                    array_layer_count: None,
+                    array_layer_count: Some(1),
```

Comment:
What difference does this make?

Reply:
Nothing really. A little bit more explicit and maybe saves a teeny bit of wgpu validation.